projects/packages/videopress/src/class-initializer.php

```diff
@@ -95,6 +95,10 @@ private static function unconditional_initialization() {
 		if ( is_admin() ) {
 			AJAX::init();
 		}
+
+		if ( ! is_admin() ) {
+			Block_Replacement::init();
```

Comment:
Just for my understanding. Do you know why the init is also called here?

Reply:
This initializer is used in the VideoPress plugin, but is not used in Jetpack plugin.
So I needed to call the init method from two places and make sure the filter is added only once.

Reply:
Got it, thank you!